dalli.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest', '>= 4.2.0'
   s.add_development_dependency 'mocha'
   s.add_development_dependency 'rails', '~> 4'
-  s.add_development_dependency 'rake'
+  s.add_development_dependency 'rake', '~> 10.5'
```

Comment:
Ah, excellent. Would you like me to set it to `>= 11.1.1`, or should I revert the change entirely?